examples/quickstarts/dancer-mysql.json

```diff
@@ -201,9 +207,9 @@
                   }
                 ],
                 "resources": {
-                  "limits": {
-                    "memory": "${MEMORY_LIMIT}"
-                  }
+		      "limits": {
+			  "memory": "${MEMORY_LIMIT}"
+		      }
```

Comment:
i just fixed this spacing issue upstream, can you rerun the update script?  (sorry)